modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java

```diff
@@ -1379,6 +1382,107 @@ public void testCacheDestroy() throws IgniteCheckedException {
         assertTrue("Caches must be destroyed: " + crd.cacheNames().toString(), crd.cacheNames().isEmpty());
     }
 
+    /** */
+    @Test
+    public void testCacheClear() {
```

Comment:
Please check that [GridCommandHandlerPasscodeAuthenticatorClusterByClassTest](https://github.com/ggprivate/ggprivate/blob/master/modules/core/src/test/java/org/gridgain/grid/util/GridCommandHandlerPasscodeAuthenticatorClusterByClassTest.java) passed too.
It requires updating [GridEntSecurityProcessor](https://github.com/ggprivate/ggprivate/blob/master/modules/core/src/main/java/org/gridgain/grid/internal/processors/security/GridEntSecurityProcessor.java)

Reply:
I'm not sure what we are looking for here. The security processor has visor cache clear already in it.

Reply:
Yes, the change you propose [PullRequest#1923](https://github.com/ggprivate/ggprivate/pull/1923) is correct.
Also, you need to provide additional permissions in the test for `cache1`, `cache2` and `cache3` I think

Reply:
I've adjusted the test to meet the existing permissions.
I also updated the expected output of `control.sh --cache help` to pass the testCacheHelp tests.